tutorials/notebooks/specutils-fit-line/specutils-fit-line1.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plt.plot(np.power(10.0, qso0008_coadd_data['loglam']), qso0008_coadd_data['flux'])
</pre><p>I kinda want to be picky and add an extra line to create a new variable called lambda rather than including the conversion int the call to plt.</p><p><code>lambda = np.power(10.0,</code>so0008_coadd_data['loglam'])<code> </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I normally prefer to take advantage of "the power of OOP" to pull data directly from the object, rather than setting up a new variable name.</p><p> </p><p>But since it's cluttering up the plot call, I'm fine with doing it in this particular case. Changing this now.</p>